docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb

```diff
@@ -186,13 +186,16 @@
       "template<>\r\n",
       "void Dummy<::dali::GPUBackend>::RunImpl(::dali::DeviceWorkspace &ws) {\r\n",
       "  const auto &input = ws.Input<::dali::GPUBackend>(0);\r\n",
+      "  const auto &shape = input.shape();\r\n",
       "  auto &output = ws.Output<::dali::GPUBackend>(0);\r\n",
-      "  CUDA_CALL(cudaMemcpyAsync(\r\n",
-      "          output.raw_mutable_data(),\r\n",
-      "          input.raw_data(),\r\n",
-      "          input.nbytes(),\r\n",
-      "          cudaMemcpyDeviceToDevice,\r\n",
-      "          ws.stream()));\r\n",
+      "  for (int sample_idx = 0; sample_idx < shape.num_samples(); sample_idx++) {\r\n",
+      "    CUDA_CALL(cudaMemcpyAsync(\r\n",
```

Comment:
Is this due to rework towards non-contiguous TL/TV memory?

Reply:
Looks like it wasn't regenerated after that change.